models/programs.js

```diff
@@ -17,6 +17,16 @@ module.exports = {
         type : String,
         index : true
       },
+      startDate:{
```

Comment:
@borkarsaish65 Hence we updated model with new keys, It is ideal to update the model in other services as well.

Reply:
done